src/app/(explore)/topics/page.tsx

```diff
@@ -0,0 +1,16 @@
+"use client";
```

Comment:
I think this page should also be server rendered

Reply:
same issues https://github.com/bitcointranscripts/registry/pull/43#discussion_r1795509650

Reply:
same as https://github.com/bitcointranscripts/registry/pull/43#discussion_r1795509650